lib/DonorsList/DonorsList.js

```diff
@@ -0,0 +1,107 @@
+import React, { useMemo } from 'react';
+import { sortBy } from 'lodash';
+import PropTypes from 'prop-types';
+import { useIntl } from 'react-intl';
+
+import {
+  Button,
+  Icon,
+  MultiColumnList,
+} from '@folio/stripes/components';
+import { useStripes } from '@folio/stripes/core';
+
+import { acqRowFormatter } from '../utils';
+import AddDonorButton from './AddDonorButton';
+import { alignRowProps, columnMapping, columnWidths, visibleColumns } from './constants';
+
+const getResultsFormatter = ({
+  intl,
+  fields,
+}) => ({
+  name: donor => donor.name,
+  code: donor => donor.code,
+  unassignDonor: (donor) => (
```

Comment:
Please keep consistency in code style: two args in the callbacks before not wrapped in the `(..)`